frame/nomination-pools/src/lib.rs

```diff
@@ -1466,7 +1450,7 @@ pub mod pallet {
 			let who = ensure_signed(origin)?;
 			let mut bonded_pool = BondedPool::<T>::get(pool_id).ok_or(Error::<T>::PoolNotFound)?;
 			ensure!(bonded_pool.state != PoolState::Destroying, Error::<T>::CanNotChangeState);
-			// TODO: [now] we could  check if bonded_pool.ok_to_be_open().is_err(), and if thats
+			// TODO: [now] we could check if bonded_pool.ok_to_be_open().is_err(), and if thats
```

Comment:
@kianenigma do you have an opinion on if we should do this? The only reason I would be against this is if the `ok_to_be_open` checks end up being wrong and we accidentally close some pools - but ATM I am confident this would not happen